.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,46 @@
+name: End-to-end (e2e) Tests
+
+on:
+  # Run on all pushes and on all pull requests.
+  # Prevent the "push" build from running when there are only irrelevant changes.
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: e2e against WordPress latest
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Spin up WordPress
+        run: docker-compose -f tests/e2e/docker-compose.yml up &
```

Comment:
What does the trailing `&` do here?
(It's not included in the readme for manual running the tests.)

Reply:
I've tested the removal of it and it breaks the GitHub Actions workflow.